.github/workflows/build-and-release.yml

```
@@ -286,11 +323,61 @@ jobs:
               /tmp/opennox-win-release/OpenNox-*.exe
             if-no-files-found: error
 
+  release_repackage_win7:
+      name: Repackage Windows release (Win7 compatible)
+      runs-on: ubuntu-latest
+      if: startsWith(github.ref, 'refs/tags/') || (github.ref == 'refs/heads/ci_tests')
+      needs: [hybrid_docker_win7]
+      steps:
+        - name: Install NSIS
+          run: sudo apt-get install -y nsis
+
+        - uses: actions/checkout@v3
+          with:
+            fetch-depth: 0
+            submodules: true
+
+        - name: Cleanup
+          run: |
+            rm -rf /tmp/win-tmp
+            mkdir -p /tmp/win-tmp
+
+        - uses: actions/download-artifact@v3
+          with:
+            name: opennox-win7
+            path: /tmp/win-tmp/
+
+        - name: Repackage release
+          run: |
+            ls -lh /tmp/win-tmp/
+            rm -rf /tmp/opennox-win
+            mkdir -p /tmp/opennox-win
+            cp /tmp/win-tmp/*.exe /tmp/opennox-win/
+            cp /tmp/win-tmp/*.dll /tmp/opennox-win/
+            cp ./CHANGELOG.md /tmp/opennox-win/
+
+            rm -rf /tmp/opennox-win-release
+            mkdir -p /tmp/opennox-win-release
+            export VERSION=$(git name-rev --tags --name-only $(git rev-parse HEAD))
+            makensis -DVERSION="$VERSION-win7" -DDATA="/tmp/opennox-win/" -DOUTDIR="/tmp/opennox-win-release/" ./scripts/opennox.nsi
+
+            cd /tmp/opennox-win
+            zip -9 -r /tmp/opennox-win-release/OpenNox-$VERSION-win7.zip ./*
```

```suggestion
            zip -9 -r /tmp/opennox-win-release/OpenNox-win7-$VERSION.zip ./*
```
Since other releases put version at the end.